### PR TITLE
Add MintException exception class

### PR DESCRIPTION
**Pull request by titilambert**
This PR create a new Exception class, called MintException.
This is useful when you use mintapi as a librairy, in order to not catch all exceptions.

### CODE REVIEW

File: mintapi/api.py [hunk 1]
```diff
@@ -171,7 +175,7 @@ def get_session_cookies(self, username, password):
         try:
             from selenium import webdriver
             driver = webdriver.Chrome()
-        except Exception as e:
+        except MintException as e:
```
Comment (mrooney) [inline mintapi/api.py]: I don't believe this should be MintException. Other python exceptions (ImportError among others) could be raised and we want to catch them all!

Comment (titilambert) [inline mintapi/api.py]: Ho ! You're right !

**Comment by titilambert** [comment]
Change done !

**Comment by mrooney** [comment]
Thank you very much for your contribution!

**Comment by titilambert** [comment]
Thank you !
Do you have an ETA for the next release ?
I would like to use the next release in the PR: https://github.com/home-assistant/home-assistant/pull/6132

**Comment by mrooney** [comment]
Released today as 1.23! :)

**Comment by titilambert** [comment]
Thanks !

**Pull request merged**